src/app/static/src/app/components/projects/ProjectHistory.vue

```diff
@@ -377,9 +377,10 @@
             createProject: mapActionByName(namespace, "createProject"),
             getProjects: mapActionByName(namespace, "getProjects"),
             versionCountLabel(project: Project) {
+                const lng = this.currentLanguage;
                 return project.versions.length == 1
-                    ? "1 version"
-                    : `${project.versions.length} versions`;
+                    ? `1 ${i18next.t("versionCountLabelSingle", {lng})}`
+                    : `${project.versions.length} ${i18next.t("versionCountLabelPlural", {lng})}`;
```

Comment:
Again, translating '1 version'/'2 versions' wasn't necessary in French, but is in Portuguese. Also, 'versions' in Portuguese isn't just the world for 'version' with an s on the end, so we do need separate translations for singular and plural..